CMakeLists.txt

```diff
@@ -160,6 +160,29 @@ if(DOXYGEN_FOUND)
     VERBATIM)
 endif()
 
+# =============================================================================
+# Setup Sphinx documentation
+# =============================================================================
+add_custom_target(
+        travisDocs
```

Comment:
just to be very `nitpicky` : historically Fernando named scripts as `travis_docs`.we can just use generic target name (e.g. just docs or prepare_docs; snakecase :p ).